rewrite-java/src/main/java/org/openrewrite/java/tree/J.java

```diff
@@ -4273,6 +4273,11 @@ final class ParameterizedType implements J, TypeTree, Expression {
         @Nullable
         JContainer<Expression> typeParameters;
 
+        @With
+        @Nullable
+        @Getter
+        JavaType type;
```

Comment:
Wouldn't this always be a `JavaType.Parameterized` ?

Reply:
Good point, yes. I could have a manually written with (to conform with inherited the API) doing the cast.

Reply:
Changing the field type (and thus also the type of the wither, getter and constructor parameter) doesn't really work, since we also have to be able to deal with `JavaType.Unknown`.

Reply:
Fair enough